integreat_cms/api/urls.py

```diff
@@ -123,6 +124,16 @@
 urlpatterns = [
     path("api/regions/", include(region_api_urlpatterns)),
     path("wp-json/extensions/v3/sites/", include(region_api_urlpatterns)),
+    path(
+        "api/social/<slug:region_slug>/<slug:language_slug>/",
```

Comment:
[At least for google](https://developers.google.com/search/docs/appearance/site-names?hl=de#technical-guidelines) the root page (https://integreat.app) also needs the correct title and (maybe also some other configuration?). I think this would need separate handling.

Reply:
True, we need fallback strings.